app/controllers/timecards_controller.rb

```diff
@@ -40,6 +40,28 @@ def update
     end
   end
 
+  def verify
+    authorize! :edit, Timecard
+    @timecard = Timecard.find(params[:timecard_id])
```

Comment:
If we added Verify to the Ability class as mentioned above, we would not need lines 44-45. Also, I don't think 44 will work the way we want, as it authorizing on the class Timecard, not an instance of timecard. Removing 45 will also eliminate the Code Climate complaint as well.